src/com/projectkorra/projectkorra/firebending/lightning/Lightning.java

```diff
@@ -139,8 +141,9 @@ public Lightning(final Player player) {
 	public void electrocute(final LivingEntity lent) {
 		playLightningbendingSound(lent.getLocation());
 		playLightningbendingSound(this.player.getLocation());
+		playLightningbendingHitSound(lent.getLocation());
+		playLightningbendingHitSound(this.player.getLocation());
 		DamageHandler.damageEntity(lent, this.damage, this);
-
 		if (Math.random() <= this.stunChance) {

```

Comment:
Might as well change this and other instances of `Math.random()` to use `ThreadLocalRandom.current().nextDouble()`